services/dmx.md

```diff
@@ -0,0 +1,30 @@
+#  DMX
+
+A service that communicates using DMX512-A protocol. More at https://en.wikipedia.org/wiki/DMX512.
+
+    identifier: 0x11cf8c05
+    camel: dmx
+
+A DMX universe contains 512 channels stored as bytes. This is too large to expose as a single register. 
+Are pipes the way to go?
+
+## Registers
+
+    rw enabled?: bool @ 0x80
+
+Determines if the DMX bridge is active
+
+    rw terminate?: bool @ 0x80
```

Comment:
I don't think the controller (DMX master) ever terminates; there could a terminate switch on some of the slaves, but they would not talk JACDAC likely